src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageCategoryType.php

```diff
@@ -34,19 +34,23 @@
 use PrestaShopBundle\Form\Admin\Type\ShopChoiceTreeType;
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
 use PrestaShopBundle\Form\Admin\Type\TranslatableType;
-use PrestaShopBundle\Translation\TranslatorAwareTrait;
-use Symfony\Component\Form\AbstractType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
 use Symfony\Component\Validator\Constraints\Length;
 use Symfony\Component\Validator\Constraints\NotBlank;
 
 /**
  * Defines form for Improve > Design > Pages > Categories create/edit actions
  */
-class CmsPageCategoryType extends AbstractType
+class CmsPageCategoryType extends TranslatorAwareType
 {
-    use TranslatorAwareTrait;
+    private const NAME_MAX_LENGTH = 64;
+    private const FRIENDLY_URL_MAX_LENGTH = 64;
+    private const META_TITLE_MAX_LENGTH = 255;
+    private const META_KEYWORDS_MAX_LENGTH = 255;
+    private const META_DESCRIPTION_MAX_LENGTH = 512;
 
```

Comment:
I have added constants for max length values. While META_TITLE_MAX_LENGTH  and META_KEYWORDS_MAX_LENGTH have the same values, I can't find a connection between them, so I added 2 different constants.

Reply:
I see no reason to make these consts as private :thinking:

Reply:
The class is not final, it can be extended, it can also be used in a template, I recommend putting these kinds of consts in public

Reply:
Good point made them public.